src/status_im/ui/screens/home/animations/responder.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im.ui.screens.home.animations.responder
+  (:require [status-im.ui.components.react :as react]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.animation :as animation]))
+
+(defn get-updated-value [gesture end-offset-x swiped?]
+  (let [base-value (if swiped? end-offset-x 0)]
+    (- base-value (.-dx gesture))))
+
+(defn on-start [_ gesture]
+  (> (Math/abs (.-dx gesture)) 10))
```

Comment:
Wouldn't `js/Math.abs` be more idiomatic?

Reply:
In this project, there is two place use `Math/abs`, and no occurrence of `js/math.abs` yet.

Reply:
Those are probably outdated: proper ClojureScript interop syntax for external calls is to use `js/`